addons/web/static/src/boot.js

```diff
@@ -460,28 +149,68 @@
             alert.appendChild(
                 list(
                     "The following modules failed to load because of an error, you may find more information in the devtools console:",
-                    failed
-                )
-            );
-            alert.appendChild(
-                list(
-                    "The following modules could not be loaded because they form a dependency cycle:",
-                    cycle && [cycle]
-                )
-            );
-            alert.appendChild(
-                list(
-                    "The following modules are needed by other modules but have not been defined, they may not be present in the correct asset bundle:",
-                    missing
```

Comment:
I feel like we probably want to keep reporting about missing dependencies and cycles

Reply:
of course, it's just not done yet. this is a WIP